src/prefect/serialization/storage.py

```diff
@@ -61,6 +61,25 @@ def create_object(self, data: dict, **kwargs: Any) -> Docker:
         return base_obj
 
 
+class S3Schema(ObjectSchema):
+    class Meta:
+        object_class = S3
+
+    aws_access_key_id = fields.String(allow_none=True)
+    aws_secret_access_key = fields.String(allow_none=True)
+    aws_session_token = fields.String(allow_none=True)
+    bucket = fields.String(allow_none=True)
```

Comment:
Won't the user need to specify a bucket name for all of this to work?

Reply:
Yeah, see comment above